Makefile.am

```diff
@@ -372,6 +372,16 @@ perf_inproc_lat_SOURCES = perf/inproc_lat.cpp
 
 perf_inproc_thr_LDADD = src/libzmq.la
 perf_inproc_thr_SOURCES = perf/inproc_thr.cpp
+
+if ENABLE_STATIC
+noinst_PROGRAMS += \
+	perf/benchmark_radix_tree
+
+perf_benchmark_radix_tree_CPPFLAGS = -I$(top_srcdir)/src
+perf_benchmark_radix_tree_LDADD = $(top_builddir)/src/.libs/libzmq.a \
+	${src_libzmq_la_LIBADD}
+perf_benchmark_radix_tree_SOURCES = perf/benchmark_radix_tree.cpp
```

Comment:
to finally fix the build (i've tested it), add:

```
perf_benchmark_radix_tree_DEPENDENCIES = src/libzmq.la
```

For some reasons automake is not picking up the dependency automatically, so let's just add it manually.

Reply:
The unit tests also have this dependency though, shouldn't there be a problem with those too? Maybe because they are built after `tests`, which needs `libzmq.la`?

Reply:
Yes, probably the tests target is always built after the programs/libraries targets, I'd assume (but I really really don't want to go and look into automake to confirm...)